modules/programs/newsboat.nix

```
@@ -68,7 +68,7 @@ in {
           url = "http://example.com";
           tags = [ "foo" "bar" ];
         }];
-        description = "List of news feeds.";
+        description = "List of news feeds. Leave it empty if you want to manage feeds imperatively, e.g. using Syncthing.";
```

Best to avoid excessively long lines. For example,
```suggestion
        description = ''
          List of news feeds. Leave it empty if you want to manage feeds
          imperatively, for example, using Syncthing.
        '';
```